repository/repository_test.go

```diff
@@ -0,0 +1,30 @@
+package repository
+
+import (
+	"github.com/cloudogu/cesapp-lib/core"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNewRemoteDoguDescriptorRepository(t *testing.T) {
+	remoteConfig := &core.Remote{}
+	credentials := &core.Credentials{}
+	got, err := NewRemoteDoguDescriptorRepository(remoteConfig, credentials)
+
+	assert.NotNil(t, got)
+	assert.Nil(t, err)
+}
+
+func TestNewRemoteDoguDescriptorRepository_WithProxySettingsEnabled(t *testing.T) {
+	remoteConfig := &core.Remote{
+		ProxySettings: core.ProxySettings{
+			Enabled:  true,
+			Username: "Test",
+		},
+	}
+	credentials := &core.Credentials{}
+	got, err := NewRemoteDoguDescriptorRepository(remoteConfig, credentials)
+
+	assert.NotNil(t, got)
```

Comment:
what purpose serves this test over the othe one in this file if just "NotNil" is testet?